webapp_utils/webapp_utils.py

```diff
@@ -30,4 +31,4 @@ def set_default_headers(response): # pylint: disable=unused-variable
 application = create_app() # pylint: disable=invalid-name
 
 if __name__ == "__main__":
-    application.run(port=8083)
+    application.run(host='0.0.0.0', port=8083)
```

Comment:
I see also, that there is `health` function in utilities.py module. 
1. If it is handler called through GET /v1/health, it should be called from app. Now it's not.
2. I would prefer module utils.py rather than utilities.py, as it's simpler. But it's more common to store several frequently used and handy functions in such module, not web handler method.